ledger/metrics/src/main/scala/com/daml/metrics/IndexedUpdatesMetrics.scala

```diff
@@ -21,8 +21,9 @@ class IndexedUpdatesMetrics(prefix: MetricName, metricFactory: MetricsFactory) {
   )
 
   @MetricDoc.Tag(
-    summary = "Updates processed by the indexer",
-    description = "Represents the total number of updates processed, that are sent for indexing.",
+    summary = "Number of transactions processed.",
+    description =
+      "Represents the total number of transaction acceptance, transaction rejection, package upload, party allocation, etc. processed.",
     qualification = Debug,
   )
   val eventsMeter: MetricHandle.Meter =
```

Comment:
I was wondering if we wanted to align the Prometheus descriptions with auto-generated summaries (i.e., `MetricDoc.Tag` annotations) in the metrics doc?

Reply:
Yes, good point.